documentation/tutorial/05-deploy-admin.md

```diff
@@ -7,6 +7,14 @@ Please refer to this page for more information and the overview of the content.
 The Temporal Admin Tools are a set of command-line utilities used to configure
 and support the Temporal server.
 
+## Requirements
+
+- You have completed [Deploy PostgreSQL Database](./04-deploy-db.md).
+- A controller, `temporal-controller`, and a model, `temporal-model`, are available.
+- The application `temporal-k8s` (Temporal Server) is deployed to `temporal-model`.
+- The application `postgresql-k8s` (PostgreSQL) is deployed to `temporal-model`.
+- `temporal-k8s` is related to `postgresql-k8s` on the `db` and `visibility` endpoints.
```

Comment:
nit: consider replace `on` with `using` (using the `db` and `visibility` endpoints)